[2025-06-25T03:07:41.299+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-25T03:07:41.357+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: clinlytix360_pipeline.run_etl_cleaning manual__2025-06-25T03:07:27.304190+00:00 [queued]>
[2025-06-25T03:07:41.375+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: clinlytix360_pipeline.run_etl_cleaning manual__2025-06-25T03:07:27.304190+00:00 [queued]>
[2025-06-25T03:07:41.377+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-06-25T03:07:41.410+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): run_etl_cleaning> on 2025-06-25 03:07:27.304190+00:00
[2025-06-25T03:07:41.431+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=237) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-25T03:07:41.460+0000] {standard_task_runner.py:63} INFO - Started process 239 to run task
[2025-06-25T03:07:41.439+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'clinlytix360_pipeline', 'run_etl_cleaning', 'manual__2025-06-25T03:07:27.304190+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/clinlytix_dag.py', '--cfg-path', '/tmp/tmp4bwbb76b']
[2025-06-25T03:07:41.468+0000] {standard_task_runner.py:91} INFO - Job 27: Subtask run_etl_cleaning
[2025-06-25T03:07:41.871+0000] {task_command.py:426} INFO - Running <TaskInstance: clinlytix360_pipeline.run_etl_cleaning manual__2025-06-25T03:07:27.304190+00:00 [running]> on host 34096c58d0ef
[2025-06-25T03:07:42.996+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ajayc' AIRFLOW_CTX_DAG_ID='clinlytix360_pipeline' AIRFLOW_CTX_TASK_ID='run_etl_cleaning' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T03:07:27.304190+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-25T03:07:27.304190+00:00'
[2025-06-25T03:07:43.007+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-25T03:07:43.013+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-25T03:07:43.018+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/data/scripts/etl/etl_cleaning.py']
[2025-06-25T03:07:43.148+0000] {subprocess.py:86} INFO - Output:
[2025-06-25T03:07:49.093+0000] {subprocess.py:93} INFO - 
[2025-06-25T03:07:49.103+0000] {subprocess.py:93} INFO - ðŸ©º Cleaning ehr_oncology.csv...
[2025-06-25T03:07:49.106+0000] {subprocess.py:93} INFO - âœ… Survival days match follow-up correctly.
[2025-06-25T03:07:49.108+0000] {subprocess.py:93} INFO - 
[2025-06-25T03:07:49.110+0000] {subprocess.py:93} INFO - ðŸ“‹ Cleaning trial_protocol.csv...
[2025-06-25T03:07:49.115+0000] {subprocess.py:93} INFO - âœ… Trial protocol cleaned successfully.
[2025-06-25T03:07:49.119+0000] {subprocess.py:93} INFO - 
[2025-06-25T03:07:49.124+0000] {subprocess.py:93} INFO - ðŸ§  Cleaning pro_scores.csv...
[2025-06-25T03:07:49.126+0000] {subprocess.py:93} INFO - âœ… PRO scores clean.
[2025-06-25T03:07:49.132+0000] {subprocess.py:93} INFO - 
[2025-06-25T03:07:49.136+0000] {subprocess.py:93} INFO - âœ… All datasets cleaned and saved successfully.
[2025-06-25T03:07:49.538+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-25T03:07:49.545+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-25T03:07:49.638+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=clinlytix360_pipeline, task_id=run_etl_cleaning, execution_date=20250625T030727, start_date=20250625T030741, end_date=20250625T030749
[2025-06-25T03:07:49.723+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-25T03:07:49.823+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-06-25T03:07:49.832+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
